guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -10,7 +10,7 @@ endif::[]
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
 ifdef::load-balancing[]
-. From the *{SmartProxy}* dropdown list, select the load balancer.
+. From the *{SmartProxy}* dropdown list, select a {SmartProxyServer} configured for load balancing.
```

Comment:
```suggestion
. From the *{SmartProxy}* dropdown list, select your {SmartProxyServer} configured for load balancing.
```

suggestion to avoid "a orcharhino Proxy Server".